app/src/main/java/net/skyscanner/backpack/demo/data/BpkBarChartData.kt

```diff
@@ -77,16 +80,13 @@ object BpkBarChartData {
   fun createBar(
     date: LocalDate,
     locale: Locale = Locale.getDefault(),
-    badge: String? = null,
-    value: Float = 0.5f,
-    inactive: Boolean = false,
+    values: BpkBarChartModel.Values? = null,
```

Comment:
singular value? it's still referring to a single value, represented in two different forms